.env.local.default

```diff
@@ -200,4 +200,8 @@ DEFAULT_USER_SCOPES=editor:write,location:read,location:write
 BLOCKCHAIN_URL=http://localhost:8080/api/v1
 BLOCKCHAIN_URL_SECRET=secret
 
-FRONTEND_SERVICE_URL=https://local-matchmaking.xrengine.io/v1/frontendservice
\ No newline at end of file
+FRONTEND_SERVICE_URL=https://local-matchmaking.xrengine.io/v1/frontendservice
+
+#define logging url
+ELASTIC_HOST_DEV=http://localhost:9200
```

Comment:
Let's call this `ELASTIC_HOST_LOCAL` -- dev is actually a separate cluster located at https://dev.theoverlay.io/

Reply:
we should only need one env var `ELASTIC_HOST` since these contexts are all configured separately